cmd/hyperkube/hyperkube.go

```diff
@@ -178,7 +178,7 @@ func (hk *HyperKube) Run(args []string) error {
 
 	err = s.Run(s, s.Flags().Args())
 	if err != nil {
-		hk.Println("Error:", err)
+		hk.Println("Error:", err, "\n")
```

Comment:
Same here. `Println` should already include an newline at the end. This is not needed.